src/overlays/actors/ovl_En_Bomjimb/z_en_bomjimb.c

```diff
@@ -651,12 +651,11 @@ void func_80C02704(EnBomjimb* this, GlobalContext* globalCtx) {
     }
 }
 
+u16 D_80C03230[] = {
+    0x0721, 0x0722, 0x0723, 0x0724, 0x072C,
+};
 void func_80C02740(EnBomjimb* this, GlobalContext* globalCtx) {
```

Comment:
I think it's okay for this to stay in the function.

Reply:
What's the motivation for keeping this in the function? I thought we were trying to keep these out of the function so we could remove the static for the map file? I actually have no preference, so I'm good to move it back into the function. Just curious.

Reply:
When it's short simple data like this it doesn't matter as much. It's bss that we cannot have as static.